### PR TITLE
Fixed test and enabled it

### PR DESCRIPTION
**Pull request by carloscaldas**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/666)

### CODE REVIEW

**Comment by carloscaldas** [comment]
test!

File: src/main/scala/beam/sim/BeamMobsim.scala [hunk 1]
```diff
@@ -164,9 +164,10 @@ class BeamMobsim @Inject()(
         )
         context.watch(rideHailManager)
 
-        beamServices.vehicleTypes.get(
-          Id.create(beamServices.beamConfig.beam.agentsim.agents.rideHail.vehicleTypeId, classOf[BeamVehicleType])
-        ) match {
+        private val value: Id[BeamVehicleType] = Id
```
Comment (JustinPihony) [inline src/main/scala/beam/sim/BeamMobsim.scala]: Could you rename this to be more meaningful?

File: src/test/scala/beam/agentsim/SingleModeSpec.scala [hunk 2]
```diff
@@ -62,8 +63,24 @@ class SingleModeSpec
     beamConfig = BeamConfig(config)
 
     // Have to mock a lot of things to get the router going
+    //    val services: BeamServices =
```
Comment (JustinPihony) [inline src/test/scala/beam/agentsim/SingleModeSpec.scala]: Can you remove these commented lines as it's just potentially confusing white noise.

File: test/input/sf-light/sf-light.conf [hunk 3]
```diff
@@ -34,8 +34,8 @@ beam.agentsim.agents.modalBehaviors.lccm.paramFile = ${beam.inputDirectory}"/lcc
 beam.agentsim.agents.vehicles.bicycles.useBikes=false
 #BeamVehicles Params
 beam.agentsim.agents.vehicles.beamFuelTypesFile = ${beam.inputDirectory}"/beamFuelTypes.csv"
-beam.agentsim.agents.vehicles.beamVehicleTypesFile = ${beam.inputDirectory}"/vehicleTypes.csv"
-beam.agentsim.agents.vehicles.beamVehiclesFile = ${beam.inputDirectory}"/vehicles.csv"
+beam.agentsim.agents.vehicles.beamVehicleTypesFile = ${beam.inputDirectory}"/sample/1k/vehicleTypes.csv.gz"
```
Comment (JustinPihony) [inline test/input/sf-light/sf-light.conf]: Were these left in from other testing, or should these be kept as their normal default?

Comment (carloscaldas) [reply test/input/sf-light/sf-light.conf]: The previous file name was incompatible with household file

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Pull request merged**